conf/psmn.config

```diff
@@ -13,49 +13,44 @@ charliecloud {
 }
 
 process {
-    resourceLimits = [
-        memory: 370.GB,
-        cpus: 32,
-        time: 96.h
-    ]
```

Comment:
Was this removal intended? Seems like you'd want to keep `resourceLimits` around.

Reply:
Using `clusterOptions` to select partition is not wrong, but one could consider `queue` instead to manage those needing to override `clusterOptions`.

Reply:
Oops, I missed that - yes the resource limits definitely should be included!

Reply:
Thank you for the comment, so I can replace :
```
clusterOptions = "--partition=E5,Lake"
```

by :

```
queue = "E5,Lake"
```

with slurm ?

Reply:
Yes.